apps/server/src/modules/learnroom/imscc/imscc-file-builder.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { ImsccFileBuilder } from '@src/modules/learnroom/imscc/imscc-file-builder';
+
+describe('ImsccFileBuilder', () => {
+	let builder: ImsccFileBuilder;
+
+	beforeAll(() => {
+		builder = new ImsccFileBuilder();
+	});
+
+	describe('addTitle', () => {
+		it('should return builder', () => {
+			expect(builder.addTitle('Placeholder')).toEqual<ImsccFileBuilder>(builder);
```

Comment:
It covered the code, but you should check if the title was added.

Reply:
Done.